### PR TITLE
Add tip for multiple co-dependent PRs

### PR DESCRIPTION
**Pull request by danielhaan**

### CODE REVIEW

**Review by Snailed — COMMENTED** [review]
Should we mention the relevant CLI flag?

File: how-we-use-github.md [hunk 1]
```diff
@@ -21,6 +21,21 @@ Each feature branch should have the name `feature/[issue-id]-[title]` correspond
...
+You will typically just create your new branch from the un-merged one and do your work.
+
+When you then create the second PR, edit it's base to the older PR.
+
+
+**That way you gain 2 advantages:**
+1. The reviewer of this PR doesn't accidentally review the code from the earlier PR
+2. You don't have merge conflicts when either of these are merged to the trunk
+
+The only thing you need to be careful about is, that the earlier PR is merged *before* this new one. Otherwise this PRs code diff will show up as diff in the earlier one.
```
Comment (Snailed) [inline how-we-use-github.md:36]: How do we prevent this?
Should we maybe put a comment on the PR?

**Review by Snailed — CHANGES_REQUESTED** [review]
See comments. If you do not think changes are needed, I will approve.

File: how-we-use-github.md (hunk 1 above)
Comment (danielhaan) [reply how-we-use-github.md:36]: Yeah, I can add that as a suggestion as well? There's no way to enforce people do that, but it helps

Comment (danielhaan) [reply how-we-use-github.md:36]: In general people should always add a comment to clarify when doing out-of-the-norm things

File: how-we-use-github.md [hunk 2]
```diff
@@ -21,6 +21,21 @@ Each feature branch should have the name `feature/[issue-id]-[title]` correspond
...
+Merging the earlier one first and deleting the branch will cause this PR to automatically resolve it's base branch to staging, meaning everthing will work smoothly :-)
+
```
Comment (danielhaan) [inline how-we-use-github.md:38]:
```suggestion

We recommend adding a note on the 2nd PR, so that others are aware of this special case.
```

**Comment by danielhaan** [comment]
@Snailed I added an extra line suggesting to add a note in the PRs where it's relevant

**Pull request merged**